react/features/invite/components/ShareLinkForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+
+import { translate } from '../../base/i18n';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * A React Component for displaying a value with a copy button that can be
+ * clicked to copy the value onto the clipboard.
+ */
+class ShareLinkForm extends Component {
+    /**
+     * ShareLinkForm component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: React.PropTypes.func,
+
+       /**
+         * The value to be displayed and copied onto the clipboard.
+         */
+        toCopy: React.PropTypes.string
+    }
+
+    /**
+     * Initializes a new ShareLinkForm instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     */
+    constructor(props) {
+        super(props);
+
+        this._inputElement = null;
```

Comment:
Technically, the above introduces a field and our coding style requires jsdocs on everything which includes fields. Anyway, I'll write this one in order to spare time.